instrumentation/messaging/src/test/resources/log4j2.properties

```diff
@@ -0,0 +1,15 @@
+appenders=console
+appender.console.type=Console
+appender.console.name=STDOUT
+appender.console.layout.type=PatternLayout
+appender.console.layout.pattern=%d{ABSOLUTE} %-5p [%t] %C{2} (%F:%L) - %m%n
+rootLogger.level=warn
+rootLogger.appenderRefs=stdout
+rootLogger.appenderRef.stdout.ref=STDOUT
+# uncomment to include kafka consumer configuration in test logs
```

Comment:
probably don't need this as this module doesn't have kafka inside I think.

later, we can add "messaging-tests" module with ITs in them and maybe a default log file with kafka config could be relevant.